ddp-workspace/projects/ddp-dsm-ui/src/app/field-settings/field-settings.component.html

```diff
@@ -43,9 +43,6 @@ <h4>{{selectedType.name}} Settings</h4>
           </th>
         </tr>
         <tr>
-          <th>
```

Comment:
@SimoneMaiwald is this PR related to avoid inconvenient `column_name` in `field_settings` from study stuff?

Reply:
yes. that was the reason for making that change

Reply:
Since we need `variable_name` in any case, I think it is better not to remove it from `frontend`, instead we may keep it but add check on the input that they could only enter values which follow `camelCase` or `camel_case` pattern